### PR TITLE
fix: use job obj directly

### PR DESCRIPTION
**Pull request by d2lam**
We have this information from here: https://github.com/screwdriver-cd/models/blob/master/lib/buildFactory.js#L248-L249

Related: 
https://github.com/screwdriver-cd/screwdriver/issues/1284
https://github.com/screwdriver-cd/screwdriver/issues/1257

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,85 +1,52 @@
...
 
-                    return resolve('');
-                }
+            return Promise.resolve(eventMap.join(' && '));
+        }
 
-                return reject(err);
-            })
-        );
+        return Promise.resovlve('');
```
Comment (pranavrc) [inline index.js]: `resolve`?

File: index.js [hunk 2]
```diff
@@ -1,85 +1,52 @@
...
-                return reject(err);
-            })
-        );
+        return Promise.resolve('');
     }
 
     /**
      * Gives the commands needed for publishing build cache after a build completes.
      * @method getTeardownCommand
+     * @param  {PipelineModel}  o.pipeline  Pipeline model for the build
```
Comment (pranavrc) [inline index.js:34]: This one's missing here: https://github.com/screwdriver-cd/cache-bookend/pull/5/files#diff-168726dbe96b3ce427e7fedce31bb0bcR10

File: index.js (hunk 1 above)
Comment (d2lam) [reply index.js]: oops, added a test case for that 👍

File: index.js (hunk 2 above)
Comment (d2lam) [reply index.js:34]: added

**Review by minzcmu — APPROVED** [review]

**Review by pranavrc — APPROVED** [review]
image

**Pull request merged**